### PR TITLE
Fixed transport drain

### PR DESCRIPTION
**Pull request by msoedov**
Hi, I found that SseServerProtocol is little bit out-updated

https://github.com/KeepSafe/aiohttp/issues/28

You can you reproduce the trace-back with sending large chunk of data (> 64kb).
Unfortunately I can't prove my changes with unit-tests. Btw, do you have future plans for maintaining this repository?

### CODE REVIEW

File: sse/protocol.py [hunk 1]
```diff
@@ -30,11 +30,12 @@ def __init__(self, sse_handler=None, **kwargs):
...
     @asyncio.coroutine
     def handle_request(self, request, payload):
-        handler = self.handler_class(self, request, payload)
+        response = Response(self.writer, 200)
+        handler = self.handler_class(self, request, response, payload)
         try:
             handler.validate_sse()
         except exceptions.SseException as e:
-            response = Response(self.transport, e.status)
+            response.stats = e.status
```
Comment (brutasse) [inline sse/protocol.py]: Did you mean `response.status`?

**Comment by brutasse** [comment]
I don't really have long-term plans with this repo. We'll see how it goes and maybe I'll transfer it to someone else one day :)

**Comment by msoedov** [comment]
To be hones i found your repo very useful. I building signalling server for WebRtc application. What do you think if I send you couple more pull requests?

**Comment by brutasse** [comment]
I don't mind at all! Feel free to send more :)

**Pull request merged**